app/helpers/provisioning_templates_helper.rb

```diff
@@ -13,7 +13,10 @@ def template_kind(template)
   end
 
   def permitted_actions(template)
-    actions = [display_link_if_authorized(_('Clone'), template_hash_for_member(template, 'clone_template'))]
+    actions = [
+      display_link_if_authorized(_('Clone'), template_hash_for_member(template, 'clone_template')),
+      display_link_if_authorized(_('Export'), template_hash_for_member(template, 'export'))
```

Comment:
When clicking this export link, the loading spinner remains in the title bar of the application. (Tested in Chrome, the download completes in the background.)